src/panels/lovelace/entity-rows/types.ts

```diff
@@ -0,0 +1,12 @@
+import { HomeAssistant } from "../../../types";
+
+export interface EntityConfig {
+  entity: string;
+  name: string;
+  icon: string;
+}
+
+export interface EntityRow {
+  hass: HomeAssistant;
+  config: EntityConfig;
```

Comment:
I don't think that this is correct, it's `setConfig(config: EntityConfig)`

Reply:
We also have different types of entity config, like the one for divider or section.

We should do something like this:

```ts
export interface EntityConfig { … }
export interface DividerConfig { … }
export interface SectionConfig { … }
export interface WeblinkConfig { … }
export type EntityRowConfig = EntityConfig | DividerConfig | SectionConfig | WeblinkConfig;
```

With `|` we indicate that it's one or the other.